### DISCUSSION TITLE
Matrix chat / server in place of or in addition to discord?

### DISCUSSION THREAD

**Discussion by dessalines** [framesurge/perseus#278] [General]
[Discord being a privacy nightmare](https://spyware.neocities.org/articles/discord), my projects, and most open source projects such as diesel, woodpecker, etc use Matrix chats instead of discord. 

Would it be possible to have an official Matrix chat?

**Comment by arctic-hen7** [comment]
I completely agree with this sentiment, and you may have noticed there's actually already a disabled matrix link on the Perseus website in preparation for this! Perseus is only a channel on Sycamore's Discord server right now, given the tight coupling of the projects, so I'd recommend reporting this there (I believe it's been raised before).

cc @lukechu10

**Reply by dessalines** [reply]
Understandable, thanks. I'll wait for them to chime in rather than creating a bunch of discussions in different places.

**Reply by erlend-sh** [reply]
Web-public support for Matrix is also about to come out as a stable alpha, which is a pretty significant value-add for any Discord room it bridges to.

https://github.com/matrix-org/matrix-public-archive

**Reply by lukechu10** [reply]
Sorry for the late reply. I completely agree about Discord's privacy issues. However, I don't think moving the whole community to Matrix is the right solution since it seems like most people are exclusively on Discord.

Perhaps a compromise would be to use some bridge to sync up Matrix with Discord. But at this point, I'm not sure if we're avoiding Discord's privacy issues or not since we are essentially sending all the data to Discord again.

**Reply by erlend-sh** [reply]
There’s a simple way to bridge it: https://t2bot.io/discord/

Part of the privacy issue is still solved, because the people talking via matrix don’t have to sign up for the discord service and use their less trustworthy app.

**Reply by arctic-hen7** [reply]
For me personally, the main concern isn't so much what Discord might log about the community itself, since that's all functionally public record, since anyone can join the server; the problem is the app itself, which does who-knows-what behind the scenes.

**Reply by lukechu10** [reply]
> For me personally, the main concern isn't so much what Discord might log about the community itself, since that's all functionally public record, since anyone can join the server; the problem is the app itself, which does who-knows-what behind the scenes.

Ah ok this makes sense. I'll try to play around a bit soon and see what I can do.
